patches/chromium/feat_add_set_theme_source_to_allow_apps_to.patch

```diff
@@ -0,0 +1,91 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <sattard@slack-corp.com>
+Date: Mon, 26 Aug 2019 14:32:41 -0700
+Subject: feat: add set_theme_source to allow apps to
+ override chromiums internal theme choice
+
+
+diff --git a/ui/native_theme/native_theme.cc b/ui/native_theme/native_theme.cc
+index 2370d15332c8c6c7dc7e3403b38891c885704d9f..45511a9319b0e02b9b2c115f7d87d82d35743d96 100644
+--- a/ui/native_theme/native_theme.cc
++++ b/ui/native_theme/native_theme.cc
+@@ -40,6 +40,7 @@ NativeTheme::NativeTheme()
+ NativeTheme::~NativeTheme() = default;
+ 
+ bool NativeTheme::ShouldUseDarkColors() const {
++  RETURN_IF_FORCED_THEME
+   return should_use_dark_colors_;
+ }
+ 
+diff --git a/ui/native_theme/native_theme.h b/ui/native_theme/native_theme.h
+index 70389e0245993faa2c17e9deefeb000280ef2368..4d56998a391c1c7508bb911eea0c16cdde443a34 100644
+--- a/ui/native_theme/native_theme.h
++++ b/ui/native_theme/native_theme.h
+@@ -21,6 +21,11 @@
+ #include "ui/native_theme/native_theme_export.h"
+ #include "ui/native_theme/native_theme_observer.h"
+ 
++#define RETURN_IF_FORCED_THEME do { \
++  if (theme_source() == ThemeSource::kForcedLight) return false; \
++  if (theme_source() == ThemeSource::kForcedDark) return true; } while (0)
```

Comment:
please don't add #defines to class headers. Just repeat this logic, or find an alternative approach.